File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        obj = {}
        
        for num in nums:
            obj[num] = 1 + obj.get(num,0)


        heap = []

        for item in obj.keys():
            heapq.heappush(heap, (obj[item], item))   
            if len(heap) > k:
                heapq.heappop(heap)

        res = []

        for i in range(len(heap)):
            res.append(heapq.heappop(heap)[1])

        return res

File: Data Structures & Algorithms/concatenation-of-array/submission-1.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        vector<int> ans(nums.size() * 2,0);
        int n = nums.size();
        for (int i=0; i < n; i++) {
            ans[i] = nums[i];
            ans[i+n] = nums[i];
        }

        return ans;
    }
};